src/Evolution/Initialization/Tags.hpp

```diff
@@ -3,22 +3,80 @@
 
 #pragma once
 
+#include <array>
 #include <cmath>
 #include <string>
+#include <vector>
 
 #include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "Domain/FunctionsOfTime/FunctionOfTime.hpp"
+#include "Domain/FunctionsOfTime/Tags.hpp"
+#include "Options/Options.hpp"
 #include "Time/Tags.hpp"
+#include "Utilities/Gsl.hpp"
 #include "Utilities/TMPL.hpp"
 #include "Utilities/TaggedTuple.hpp"
 
 namespace Initialization {
+namespace OptionTags {
+struct FuncOfTimeFromFile {
+  static constexpr OptionString help{
+      "Options for reading a FunctionOfTime from an H5 file"};
+};
+
+struct FuncOfTimeFile {
+  using type = std::string;
+  static constexpr OptionString help{"Path to an H5 Dat file"};
```

Comment:
I don't understand what this means. Do you mean the path to the HDF5 file or the path inside the HDF5 file to the H5 Dat subfile?